src/main/java/com/purbon/kafka/topology/model/impl/TopicImpl.java

```diff
@@ -224,12 +210,12 @@ public int partitionsCount() {
     return partitionCount;
   }
 
-  public TopologyBuilderConfig getAppConfig() {
-    return appConfig;
+  public TopologyBuilderConfig getBuilderConfig() {
```

Comment:
the the rename?

Reply:
find it as more explicit name to know where this configs come from.

Reply:
In the app, there are two kinds of configs:

* the app config, so how to do rbac or admin client to access the cluster, but as well, how to use configure internals of the app.
* The topic configuration per se, so how it will look like in the cluster once created.

Reply:
Agree. My goal is to introduce some consistency on how to name `TopologyBuilderConfig` across the source-code, as there are other configs. If `appConfig` is prefered, I'm ok with it. If you agree, I can look for usages and align on that one.